packages/example-codehub/src/controllers/health-controller.api.ts

```diff
@@ -3,18 +3,28 @@
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
 
-export const def = {
+import {OpenApiSpec} from '@loopback/core';
+
+export const def: OpenApiSpec = {
+  swagger: '2.0',
+  info: {
+    title: 'unused',
+    version: 'unused',
+  },
```

Comment:
I find it annoying that Controller specification has to include mandatory fields from the root OpenAPI spec object. I am proposing to rework `@api()` decorator to accept only definitions of paths, i.e. `spec.paths` of type `PathsObject`.

If we agree on that change, then I'll create a follow-up issue. I think it's important to resolve this before the first beta, because it's a major breaking change.